Branch-SDK/Branch-SDK/BNCLinkData.h

```diff
@@ -21,6 +21,7 @@
 @property (nonatomic, strong) NSString *stage;
 @property (nonatomic, strong) NSString *params;
 @property (nonatomic, assign) NSUInteger duration;
+@property (nonatomic, assign) NSString *ignoreFirstClick;
```

Comment:
i second this. reading through it wasnt very clear what ignoreFirstClick contained since it was treated as bool inside an if statement. could you please rename this?